src/webgpu/api/validation/capability_checks/limits/maxColorAttachmentBytesPerSample.spec.ts

```diff
@@ -4,7 +4,7 @@ import { align } from '../../../../util/math.js';
 
 import {
   kMaximumLimitBaseParams,
-  kLimitModes,
+  LimitsRequest,
```

Comment:
I feel like it's more accurate the way it is.

1. First it takes the bytesPerSample so far and adjusts it to the the alignment requirements for the next type.
2. Then it checks, given the aligned bytesPerSample, is there enough room for the next target.

The suggested way to me that logic is lost. It's instead "newBytesPerSize = magic math" if newBytesPerSize > testValue exit.

It might be 6 of 1, half dozen of another but it's not clear to me changing it makes it more understandable and not less.

Reply:
Hmm fair enough. I guess I came from the spec knowing the exact algorithm that should be applied here (https://www.w3.org/TR/webgpu/#abstract-opdef-calculating-color-attachment-bytes-per-sample) so the magic math part wasn't so much magic, but just a given operation/calculation. I think either leaving it as is, or having the magic math part with a comment linking to the spec is fine.

Reply:
I like that reasoning. I'll check it to match the spec algo